test/src-tests/index-test.js

```diff
@@ -46,6 +60,30 @@ describe('find-parent/index', function() {
         assert.isUndefined(result);
       });
 
+      it('should return undefined when no parent matches and throwOnMiss is false', function() {
+        var opts = {
+          throwOnMiss: false
+        };
+
+        var result = findParent.byMatcher(element, function() {
+          return false;
+        }, opts);
+
+        assert.isUndefined(result);
+      });
+
+      it('should throw when no parent matches and throwOnMiss is true', function() {
+        var opts = {
+          throwOnMiss: true
+        };
+
+        assert.throws(function() {
+          findParent.byMatcher(element, function() {
+            return false;
+          }, opts);
+        }, 'Expected to find parent node, but none was found.');
```

Comment:
We probably only need to test the error message once instead of specifying it every time we assert that it throws.